frontend/src/app/components/review/AddReviewModal.tsx

```diff
@@ -0,0 +1,50 @@
+import type { ModalProps } from "@/app/interfaces/ModalProps";
+import { Modal, ModalContent } from "@nextui-org/react";
+import React from "react";
+import ChevronLeftIcon from "../svgs/Calendar/ChevronLeftIcon";
+import AddReviewForm from "./AddReviewForm";
+
+const ReviewModal: React.FC<ModalProps> = ({ isOpen, onClose, size }) => {
+  return (
+    <div>
+      <Modal
+        size={size}
+        isOpen={isOpen}
+        onClose={onClose}
+        isDismissable={false}
+        closeButton={<></>}
+      >
+        <ModalContent>
+          {(onClose) => (
+            <div className="p-5">
+              <div className="mb-2 flex flex-col">
+                <div className="flex flex-row">
+                  <div
+                    className="my-1 cursor-pointer self-start"
+                    onClick={onClose}
+                  >
+                    <ChevronLeftIcon />
+                  </div>
```

Comment:
Can we use Button iconOnly for this?

Reply:
ok, i'll check and try Button iconOnly

Reply:
![image](https://github.com/framgia/sph_sunbnb_sim/assets/156734177/3875254c-c4b3-4a17-b905-74fcd06e8837)
_with iconOnly button_